drivers/scheduler/k8s/specs/postgres-csi/pso/pure-storage-class.yaml

```diff
@@ -1,7 +1,7 @@
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
-  name: pure-sc
+  name: postgres-sc
```

Comment:
rename this to pso-storage-class.yaml

Reply:
I thought since the provisioner is `pure-csi` I can keep it as `pure-storage-class`. But what you say also makes sense. Renamed it.